src/audio-service.ts

```diff
@@ -0,0 +1,27 @@
+import { writable } from 'svelte/store';
```

Comment:
issue: This file looks like a good candidate for an `audio` entity. In terms of FSD, this code should be located in `src/entities/audio/model/audio-instance.ts`. The filename can be whatever, though.